content/blog/entries/what-is-up-ccos/contents.lr

```diff
@@ -0,0 +1,29 @@
+title: What is up? - CCOS Revamp
+---
+categories:
+outreachy
+tech
+open-source
+---
+author: dhruvi16
+---
+pub_date: 2020-07-13
+---
+body:
+
+
+Well in my previous blog, I demonstrated what was my Outreachy project about. Here I will talk about my progress in the past 7 weeks.
+
+### The Set-Up -
+
+The [Creative Commons Open Source](https://opensource.creativecommons.org/) website is built using Lektor. I was not very familiar with [Lektor](https://www.getlektor.com/), so I started by going through the documentation and the official website code. I learned how awesome it is and can also be used by non-coders. I got familiar with the jinja templates and working of themes in a Lektor app. For integrating new styles from Vocabulary, I replaced templates/ folder with a theme/ folder.
```

Comment:
```suggestion
The [Creative Commons Open Source](https://opensource.creativecommons.org/) website is built using [Lektor](https://www.getlektor.com/). I was not very familiar with it, so I started by going through the documentation and the official website code. I learned how awesome it is and can also be used by non-coders. I got familiar with the [jinja templates](https://palletsprojects.com/p/jinja/) and working of themes in a Lektor app. For integrating new styles from Vocabulary, I replaced `templates/` folder with a `theme/` folder.
```
also you can link to Lektor template documentation, I think it could be useful for someone who wants to get more info